src/_assets/css/_toc.scss

```diff
@@ -0,0 +1,65 @@
+#site-toc--side {
+  $padding: map-get($spacers, 2);
+
+  display: none;
+  right: 0;
+
+  .nav { display: block; }
+  .nav-link {
+    padding-bottom: $padding;
+
+    color: theme-color('primary');
+    &.active {
+      color: theme-color('secondary'); // $gray-dark;
+      font-weight: 500;
+    }
+  }
+
+  > ul > li {
+    padding-bottom: $padding; // Extra space for top-level entries
+    li { padding-left: $padding; } // Indent subentries
+  }
+}
+
+.site-toc {
+  $line-height: normal;
+
+  @include media-breakpoint-up(xl) {
+    padding-bottom: $site-content-top-padding;
+    padding-top: $site-content-top-padding;
+
+    @at-root {
```

Comment:
(I'm not familiar with this macro but I trust you know what you're doing, especially if it's used elsewhere.)

Reply:
I'm not familiar with it either, but seems to be a standard SCSS macro:
https://sass-lang.com/documentation/file.SASS_REFERENCE.html#at-root_without__and_at-root_with_

(The file is a direct copy from Flutter and will be moved to shared).